analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -28,8 +28,10 @@ standard_arriba_file="${scratch_path}/arriba.tsv"
 standard_starfusion_file="${scratch_path}/starfusion.tsv"
 
 # general filtering parameters
-artifact_filter="GTEx_Recurrent|DGD_PARALOGS|Normal|BodyMap|ConjoinG"
+artifact_filter="GTEx_Recurrent|DGD_PARALOGS|Normal|BodyMap"
 reading_frame_filter="in-frame|frameshift|other"
+spanningFragCountFilter=100
```

Comment:
If you think the default for this project should be `100` (sounds like you do!), it makes sense to me to make the quick 1 line change to `02-fusion-filtering.R` in this PR. You can leave the changes to the shell script, too. That change, even if you change the default,  makes it a little easier to quickly track down that info in the code in my opinion!

Reply:
ok sounds good, I'll update the default values in `02-fusion-filtering.R`